tests/components/camera/test_init.py

```diff
@@ -279,8 +279,7 @@ async def test_snapshot_service(hass, mock_camera):
     mopen = mock_open()
 
     with patch("homeassistant.components.camera.open", mopen, create=True), patch(
-        "homeassistant.components.camera.os.path.exists",
```

Comment:
was this old patch doing anything? I don't see this method being used anywhere in the code even before this PR

Reply:
The code got updated but the tests never did. That’s fixed here